script.module.libMediathek2/lib/libMediathek2Listing.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+import urllib
+import urllib2
+import socket
+import sys
+import xbmc
+import xbmcplugin
+import xbmcgui
+import xbmcaddon
+import xbmcvfs
+import re
+from datetime import datetime,timedelta
+import time
+
+from libMediathek2Utils import clearString
+
+icon = xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('path')+'/icon.png').decode('utf-8')
+fanart = xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('path')+'/fanart.jpg').decode('utf-8')
+
+def addEntries(l):
+	lists = []
+	for dict in l:
```

Comment:
dict is quite a dangerous name to choose since python uses that name for dictionaries.